mmv1/third_party/terraform/resources/resource_compute_instance.go.erb

```diff
@@ -216,6 +217,15 @@ func resourceComputeInstance() *schema.Resource {
 										ForceNew:     true,
 										Description:  `A set of key/value label pairs assigned to the disk.`,
 									},
+
+									"resource_manager_tags": {
+										Type:         schema.TypeMap,
+										Optional:     true,
+										AtLeastOneOf: initializeParamsKeys,
+										Computed:     true,
```

Comment:
Why is this set to `Computed: true`? (& same for the new fields below)

Reply:
Yes, I guess this is unnecessary